x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/use_alert_suppression.test.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { renderHook } from '@testing-library/react-hooks';
+import type { Type } from '@kbn/securitysolution-io-ts-alerting-types';
+import { SuppressibleAlertRules } from '../../../../../common/detection_engine/constants';
+
+import { useAlertSuppression } from './use_alert_suppression';
+import type { RuleResponse } from '../../../../../common/api/detection_engine';
+
+jest.mock('../../../../common/hooks/use_experimental_features', () => ({
+  useIsExperimentalFeatureEnabled: (featureFlagName: string) =>
+    featureFlagName === 'alertSuppressionForIndicatorMatchRuleEnabled',
+}));
+
+describe('useAlertSuppression', () => {
+  it('should return the correct isSuppressionEnabled value if rule Type exists in SuppressibleAlertRules and Feature Flag is enabled', () => {
```

Comment:
can we add test, that return false for `THREAT_MATCH` rule, when  Feature Flag is disabled?

Reply:
[055475d](https://github.com/elastic/kibana/pull/175089/commits/055475d651850c32bc110c384914e918689b07a2)